src/pages/keyword-counter/KeywordCounter.vue

```diff
@@ -0,0 +1,224 @@
+                      <td class="border px-4 py-2">{{ group.count }}</td>
+                      <td class="border px-4 py-2">{{ group.density }}%</td>
+                    </tr>
+                  </tbody>
+                </table>
+              </div>
+              <div class="copy-button-wrapper">
+                <index @click="copyTable" class="copy-button"><span class="ml-2">Copy To Clipboard</span></index>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+</template>
+
+<script setup>
+import { ref, computed, onBeforeMount } from 'vue';
+import { MaInput, MaIcon } from "@mobileaction/action-kit";
+
+const userInput = ref('');
+const excludedWordsInputValue = ref('');
+const placeholderText = ref('Enter your text here...');
+const keywordStats = ref([]);
+const groupedKeywordStats = ref([]);
+
+onBeforeMount(() => {
+  userInput.value = localStorage.getItem('inputValue') || ''
+});
+
+const charCount = computed(() => userInput.value.length);
+
+const countKeywords = () => {
+  const words = cleanText(userInput.value).split(/\s+/);
+  const totalWords = words.length;
+  const keywordMap = {};
+
+  words.forEach((word) => {
+    keywordMap[word] = (keywordMap[word] || 0) + 1;
+  });
+
+  const keywordArray = Object.keys(keywordMap).map((keyword) => ({
+    keyword,
+    count: keywordMap[keyword],
+    density: ((keywordMap[keyword] / totalWords) * 100).toFixed(1),
+  }));
+
+  groupAndSortKeywords(keywordArray);
+};
+
+const groupAndSortKeywords = (keywordArray) => {
+  const groups = {};
+
+  keywordArray.forEach((item) => {
+    if (!groups[item.count]) {
+      groups[item.count] = {
+        count: item.count,
+        keywords: [],
+        density: 0,
+      };
+    }
+    groups[item.count].keywords.push(item.keyword);
+    groups[item.count].density += parseFloat(item.density);
+  });
+
+  groupedKeywordStats.value = Object.values(groups)
+    .map((group) => {
+      return {
+        ...group,
+        density: (group.density / group.keywords.length).toFixed(1),
+      };
+    })
+    .sort((a, b) => b.density - a.density);
+};
+
+const cleanText = (text) => {
+  const excludedWords = excludedWordsInputValue.value.split(' ').map(word => word.trim().toLowerCase());
+  return text
+    .replace(/[^\w\s]|_/g, "")
+    .replace(/\s+/g, " ")
+    .toLowerCase()
+    .split(' ')
+    .filter(word => word && !excludedWords.includes(word))
+    .join(' ');
+};
+
+const copyTable = () => {
+  const table = document.querySelector('.table-container table');
+  const rows = Array.from(table.querySelectorAll('tbody tr'))
+    .map(row => Array.from(row.querySelectorAll('td'))
+      .map(cell => cell.innerText)
+      .join(' : ')
+    )
+    .join('\n');
+
+  navigator.clipboard.writeText(rows)
+    .then(() => console.log('Table copied to clipboard'))
+    .catch(err => console.error('Failed to copy table: ', err));
+};
```

Comment:
It would be better to notify the user in this section. We can use the `alert()` function.